test/cpp/test_aten_xla_tensor.cpp

```diff
@@ -648,11 +648,13 @@ TEST_F(AtenXlaTensorTest, TestIntegerAdd) {
           torch::randint(0, 63, {2, 2}, torch::TensorOptions(type));
       torch::Tensor b =
           torch::randint(0, 63, {2, 2}, torch::TensorOptions(type));
-      torch::Tensor c = torch::add(b, 1.0);
+      at::Scalar one =
```

Comment:
Maybe, looking at `Scalar.h` a `torch.add(x, int64_t(1))` would work as well.

Reply:
Not sure I understand your suggestion here, would you rather have a ternary switch between `torch.add(x, int64_t(1))` and `torch.add(x, 1.0)`? If we did `torch.add(x, int64_t(1))` unconditionally, we'd have to rely on the promotion rule between a float tensor and an integer scalar to be a float. That'd be fine, I believe, I just wanted to be as explicit as possible here since it's test code.

Reply:
Oh, never mind. This is a multi-type test.